cvmfs/fuse_evict.h

```diff
@@ -94,6 +95,8 @@ class FuseInvalidator : SingleCopy {
    */
   atomic_int32 terminated_;
   BigVector<uint64_t> evict_list_;
+
+  static bool use_fuse_notify_invalidation_;
```

Comment:
Let's call it `g_use_fuse_notify_invalidation_` to make clear it is a global entity.